test/grub2base_test.rb

```diff
@@ -225,7 +261,7 @@
 
         context "on other TERM" do
           before do
-            ENV["TERM"] = "xterm"
+            allow(ENV).to receive(:"[]").with("TERM").and_return("xterm")
```

Comment:
I was surprised to find out that `:[]` is valid syntax. Not sure what RuboCop likes.